source/Calamari/TerraformCliExecutor.cs

```diff
@@ -153,18 +155,30 @@ void InitializePlugins()
         {
             var initParams = variables.Get(TerraformSpecialVariables.Action.Terraform.AdditionalInitParams);
             var allowPluginDownloads = variables.GetFlag(TerraformSpecialVariables.Action.Terraform.AllowPluginDownloads, true);
+            string initCommand = $"init -no-color {initParams}";
+
+            if (version.CompareTo(new Version("0.15.0")) < 0)
+            {
+                initCommand = $"init -no-color -get-plugins={allowPluginDownloads.ToString().ToLower()} {initParams}";
+            }
 
             ExecuteCommandInternal(
-                                   new[] { $"init -no-color -get-plugins={allowPluginDownloads.ToString().ToLower()} {initParams}" },
+                                   new[] { initCommand },
                                    out _,
                                    true)
                 .VerifySuccess();
         }
 
-        void LogVersion()
+        Version GetVersion()
         {
-            ExecuteCommandInternal(new[] { "--version" }, out _, true)
+            ExecuteCommandInternal(new[] { "--version" }, out string version, true)
                 .VerifySuccess();
+
+            version = version.Replace("Terraform v", "");
+            if (version.IndexOf('\n') != -1)
```

Comment:
Should we use `Environment.Newline` here?

Reply:
Interestingly enough it works with `\n` in all OSes but not with `Environment.Newline` (at least in Win it's defined as `\r\n`)

Reply:
Could it be that Calamari is already performing the substitution?